tests/dialects/test_transform.py

```diff
@@ -76,10 +75,198 @@ def test_tileop_init():
         static_sizes=static_sizes,
     )
 
-    printer = Printer()
-    printer.print(tile_op)
     expected = """
-%0, %1 = "transform.structured.tile_using_for"(%2) <{"static_sizes" = array<index: 8, 8>}> : (!transform.any_value) -> (!transform.any_op, !transform.any_op)
-"""
+%0, %1, %2 = "transform.structured.tile_using_for"(%3) <{"static_sizes" = array<index: 8, 8>}> : (!transform.any_value) -> (!transform.any_op, !transform.any_op, !transform.any_op)"""
     assert_print_op(tile_op, expected, None)
     assert isinstance(tile_op.results[0], SSAValue)
+
+
+def test_get_consumer_of_result():
+    result_number = 0
+    target = test.TestOp(result_types=[transform.AnyOpType()]).results[0]
+    op = transform.GetConsumerOfResult(target=target, result_number=result_number)
+    expected = """
+    %0 = "transform.get_consumers_of_result"(%1) <{"result_number" = 0 : i64}> : (!transform.any_op) -> !transform.any_op
+    """
+    assert_print_op(op, expected, None)
+    assert isinstance(op.results[0], SSAValue)
+
+
+def test_defining_op():
+    target = test.TestOp(result_types=[transform.AnyOpType()]).results[0]
+    op = transform.GetDefiningOp(target=target)
+    expected = """
+    %0 = "transform.get_defining_op"(%1) : (!transform.any_op) -> !transform.any_op    """
+    assert_print_op(
+        op,
+        expected,
+        None,
+    )
+    assert isinstance(op.results[0], SSAValue)
+
+
+def test_get_parent_op():
+    target = test.TestOp(result_types=[transform.AnyOpType()]).results[0]
+    op = transform.GetParentOp(target=target)
+    expected = """
+%0 = "transform.get_parent_op"(%1) <{"nth_parent" = 1 : i64}> : (!transform.any_op) -> !transform.any_op    """
+    assert_print_op(
+        op,
+        expected,
+        None,
+    )
```

Comment:
```suggestion
    expected = """
%0 = "transform.get_parent_op"(%1) <{"nth_parent" = 1 : i64}> : (!transform.any_op) -> !transform.any_op    """
    assert_print_op(
        transform.GetParentOp(target=target),
        """
%0 = "transform.get_parent_op"(%1) <{"nth_parent" = 1 : i64}> : (!transform.any_op) -> !transform.any_op    """,
        None,
    )
```
Probably worth doing the same throughout the file, will make it a bit easier to read